modules/nf-core/gatk4/createreadcountpanelofnormals/main.nf

```diff
@@ -0,0 +1,53 @@
+process GATK4_CREATEREADCOUNTPANELOFNORMALS {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "bioconda::gatk4=4.4.0.0"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/gatk4:4.4.0.0--py36hdfd78af_0':
+        'biocontainers/gatk4:4.4.0.0--py36hdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(counts)
+
+    output:
+    tuple val(meta), path("*.hdf5"), emit: pon
+    path "versions.yml"            , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args       = task.ext.args ?: ''
+    def prefix     = task.ext.prefix ?: "${meta.id}"
+    def input_list = counts.collect(){"--input $it"}.join(" ")
+
+    def avail_mem = 3072
+    if (!task.memory) {
+        log.info '[GATK CondenseDepthEvidence] Available memory not known - defaulting to 3GB. Specify process memory requirements to change this.'
+    } else {
+        avail_mem = (task.memory.mega*0.8).intValue()
+    }
+    """
+    gatk --java-options "-Xmx${avail_mem}M" CreateReadCountPanelOfNormals \\
+        ${input_list} \\
```

Comment:
```suggestion
        ${args} \\
        ${input_list} \\
```
Don't forget the args ;)

Reply:
Good catch! I copied this bit from another script, and I didn't realize it was missing `args` 😄